src/RestServer/RestServer.cs

```diff
@@ -0,0 +1,84 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Https;
+using Microsoft.Extensions.DependencyInjection;
+using Swashbuckle.AspNetCore.Swagger;
+using System;
+using System.Linq;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using SystemPath = System.IO.Path;
+
+namespace Neo.Plugins
+{
+    public sealed class RestServer : Plugin
+    {
+        private IWebHost host;
+
+        public override void Dispose()
+        {
+            base.Dispose();
+            if (host != null)
+            {
+                host.Dispose();
+                host = null;
+            }
+        }
+
+        protected override void Configure()
+        {
+            Settings.Load(GetConfiguration());
+        }
+
+        protected override void OnPluginsLoaded()
+        {
+            host = new WebHostBuilder().UseKestrel(options => options.Listen(Settings.Default.BindAddress, Settings.Default.Port, listenOptions =>
```

Comment:
Is not better to use the same RPC server?

Reply:
not quite sure about the meaning of the same rpc server, the rest server runs independently of rpc server